pcl_ros/CMakeLists.txt

```diff
@@ -4,7 +4,6 @@ project(pcl_ros)
 ## Find system dependencies
 find_package(cmake_modules REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system filesystem thread)
-find_package(Eigen3 REQUIRED)
```

Comment:
Please remove these changes. Eigen is very much needed for pcl_ros, and I don't understand why you're asking to remove it. Provide more context in the PR description.